gravitee-apim-console-webui/src/management/api/health-check-dashboard-v4/components/availability-per-field/availability-per-field.component.ts

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, DestroyRef, Input, OnInit } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+import { GioLoaderModule } from '@gravitee/ui-particles-angular';
+import { ActivatedRoute } from '@angular/router';
+import { switchMap, tap } from 'rxjs/operators';
+import { Observable, zip } from 'rxjs';
+import { MatSort } from '@angular/material/sort';
+import {
+  MatCell,
+  MatCellDef,
+  MatColumnDef,
+  MatHeaderCell,
+  MatHeaderCellDef,
+  MatHeaderRow,
+  MatHeaderRowDef,
+  MatRow,
+  MatRowDef,
+  MatTable,
+} from '@angular/material/table';
+import { AsyncPipe, DecimalPipe, TitleCasePipe } from '@angular/common';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+
+import { GioChartLineModule } from '../../../../../shared/components/gio-chart-line/gio-chart-line.module';
+import { ApiHealthV2Service } from '../../../../../services-ngx/api-health-v2.service';
+import { ApiAvailability, ApiAverageResponseTime } from '../../../../../entities/management-api-v2/api/v4/healthCheck';
+import { GioTableWrapperModule } from '../../../../../shared/components/gio-table-wrapper/gio-table-wrapper.module';
+import { GioTableWrapperFilters } from '../../../../../shared/components/gio-table-wrapper/gio-table-wrapper.component';
+import { gioTableFilterCollection } from '../../../../../shared/components/gio-table-wrapper/gio-table-wrapper.util';
+import { SnackBarService } from '../../../../../services-ngx/snack-bar.service';
+
+interface TableData {
+  name: string;
+  availability: number;
+  responseTime: number;
+}
+
+@Component({
+  selector: 'availability-per-field',
+  standalone: true,
+  imports: [
+    MatCardModule,
+    GioLoaderModule,
+    GioChartLineModule,
+    AsyncPipe,
+    GioTableWrapperModule,
+    MatTable,
+    MatSort,
+    MatColumnDef,
+    MatCell,
+    MatCellDef,
+    MatHeaderCell,
+    MatHeaderCellDef,
+    MatHeaderRow,
+    MatHeaderRowDef,
```

Comment:
some of the imports can be reduced by importing module containing them.

Reply:
done 👍